src/notation/internal/notationactioncontroller.cpp

```diff
@@ -100,204 +100,208 @@ void NotationActionController::init()
-    dispatcher()->reg(this, "next-chord", [this](const ActionCode& actionCode) { moveAction(actionCode); });
-    dispatcher()->reg(this, "prev-chord", [this](const ActionCode& actionCode) { moveAction(actionCode); });
-    dispatcher()->reg(this, "next-measure", [this](const ActionCode& actionCode) { moveAction(actionCode); });
-    dispatcher()->reg(this, "prev-measure", [this](const ActionCode& actionCode) { moveAction(actionCode); });
-    dispatcher()->reg(this, "next-track", [this](const ActionCode& actionCode) { moveAction(actionCode); });
-    dispatcher()->reg(this, "prev-track", [this](const ActionCode& actionCode) { moveAction(actionCode); });
-    dispatcher()->reg(this, "pitch-up", [this](const ActionCode& actionCode) { moveAction(actionCode); });
-    dispatcher()->reg(this, "pitch-down", [this](const ActionCode& actionCode) { moveAction(actionCode); });
-    dispatcher()->reg(this, "pitch-up-octave", [this](const ActionCode& actionCode) { moveAction(actionCode); });
-    dispatcher()->reg(this, "pitch-down-octave", [this](const ActionCode& actionCode) { moveAction(actionCode); });
-    dispatcher()->reg(this, "up-chord", [this]() { moveChord(MoveDirection::Up); });
-    dispatcher()->reg(this, "down-chord", [this]() { moveChord(MoveDirection::Down); });
-
-    dispatcher()->reg(this, "double-duration", [this]() { increaseDecreaseDuration(-1, /*stepByDots*/ false); });
-    dispatcher()->reg(this, "half-duration", [this]() { increaseDecreaseDuration(1, false); });
-    dispatcher()->reg(this, "inc-duration-dotted", [this]() { increaseDecreaseDuration(-1, true); });
-    dispatcher()->reg(this, "dec-duration-dotted", [this]() { increaseDecreaseDuration(1, true); });
-
-    dispatcher()->reg(this, "cut", this, &NotationActionController::cutSelection);
-    dispatcher()->reg(this, "copy", this, &NotationActionController::copySelection);
-    dispatcher()->reg(this, "paste", [this]() { pasteSelection(PastingType::Default); });
-    dispatcher()->reg(this, "paste-half", [this]() { pasteSelection(PastingType::Half); });
-    dispatcher()->reg(this, "paste-double", [this]() { pasteSelection(PastingType::Double); });
-    dispatcher()->reg(this, "paste-special", [this]() { pasteSelection(PastingType::Special); });
-    dispatcher()->reg(this, "swap", this, &NotationActionController::swapSelection);
-    dispatcher()->reg(this, "delete", this, &NotationActionController::deleteSelection);
-    dispatcher()->reg(this, "flip", this, &NotationActionController::flipSelection);
-    dispatcher()->reg(this, "tie", this, &NotationActionController::addTie);
-    dispatcher()->reg(this, "chord-tie", this, &NotationActionController::chordTie);
-    dispatcher()->reg(this, "add-slur", this, &NotationActionController::addSlur);
+    registerAction("flat2", [this]() { toggleAccidental(AccidentalType::FLAT2); });
+    registerAction("flat", [this]() { toggleAccidental(AccidentalType::FLAT); });
```

Comment:
changed so that none of these commands are available while editing text - "insert measure" was a particular problem because the default shortcut was Ctrl+B which is also for toggling bold styling. This is consistent with Mu3 and what makes logical sense anyway.